lib/bootscale/file_storage.rb

```diff
@@ -13,11 +13,13 @@ def load(load_path)
 
     def dump(load_path, cache)
       path = cache_path(load_path)
-      return if File.exist?(path)
+      raise "This should not happen" if File.exist?(path)
```

Comment:
Yes, but I think I would simply remove this line. If it's never gonna happen anymore, we might as well not do a useless `stat` call :)